draft-ietf-mls-protocol.md

```diff
@@ -948,11 +538,11 @@ A                B                C            Directory       Channel
 ~~~
 {: #prepublish-flow title="Clients A, B, and C publish KeyPackages to the directory"}
 
-{{create-flow}} shows how these prepublished KeyPackages are used to create a group.
-When a client A wants to establish a group with B and C, it first initializes a
+{{create-flow}} shows how these pre-published KeyPackages are used to create a group.
+When client A wants to establish a group with clients B and C, it first initializes a
 group state containing only itself and downloads KeyPackages for B and C. For
-each member, A generates an Add and Commit message adding that member, and
-broadcasts them to the group. It also generates a Welcome message and sends this
+each member, A generates an Add and Commit message to add that member and then
+broadcasts them to the group. Client A also generates a Welcome message and sends it
```

Comment:
Agreed, I think this terminology got carried over from the times when we had actual Add messages

Reply:
Not necessarily!  You could send an Add Proposal alongside a Commit.  At this stage, I think it is a little clearer to address them separately, with inlining as an optimization discussed later.  Reworded to be a little more explicit, though:

> A generates an Add message and a Commit message to add that member and then broadcasts the two messages to the group

Reply:
Huh?
We are describing the canonical flow here, not prescribing normative behavior. Even if we were, A sends a Commit message which contains or references A's Add proposal. The important thing for the clarity of this sentence is the Commit message, and in 99% of cases, A will never **separately** send an Add proposal. The proposed rewording misleads the reader otherwise.